prac_08/squaring.kv

```diff
@@ -0,0 +1,23 @@
+BoxLayout:
+    orientation: 'vertical'
+
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            on_press: app.handle_calculate(input_number.text)
+
+    BoxLayout:
+        Label:
+            id: output_label
+            font_size: 48
+            color: (1, 0.411, 0.705, 1)  # RGBA for pink
```

Comment:
I believe this was meant to be in the other BoxLayout with the TextInput and the Square Button.
By having it in another box layout it doesn't match the sample output we were given